src/main/java/backend/resource/TurboIssue.java

```diff
@@ -338,6 +339,28 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return Optional.empty();
     }
 
+    /**
+     * Matching is done by matching all words separated by space in query
```

Comment:
It is `AND` match because each space delimited word supposed to narrow down the suggestions right ?